pinot-integration-tests/src/test/resources/compat-tests/configs/feature-test-1-realtime.json

```diff
@@ -35,7 +35,7 @@
     "enableDefaultStarTree": false,
     "enableDynamicStarTreeCreation": false,
     "loadMode": "MMAP",
-    "noDictionaryColumns": [],
+    "noDictionaryColumns": ["textDim1"],
```

Comment:
You wont need this if you merge after this PR
https://github.com/apache/incubator-pinot/pull/6636

Reply:
Sure, will remove it if #6636 get merged first.